executor/executor.go

```diff
@@ -22,8 +22,8 @@ type Executor interface {
 }
 
 func prepareLog(smallSpec *util.CommandSpec) (err error) {
-	dirPath := filepath.Join(config.SDArtifactsDir, ".sd", "commands", smallSpec.Namespace, smallSpec.Name, smallSpec.Version)
-	filename := fmt.Sprintf("%v.log", time.Now().Unix())
+	dirPath := filepath.Join(config.SDArtifactsDir, ".sd", "commands")
+	filename := fmt.Sprintf("%v-%v-%v.log", time.Now().Unix(), smallSpec.Namespace, smallSpec.Name)
```

Comment:
Do we need `Version` information?

Reply:
I think that there are few cases where different versions of the same commands are used in the same build, therefore `Version` information has little information. Even if different versions of the same commands are used in the same build, we can distinguish them by timestamp.

Reply:
make sense.